.github/workflows/ci.yaml

```diff
@@ -1,11 +1,25 @@
 name: Python CI
 
-on: [push]
+"on":
+  push:
+    branches-ignore:
```

Comment:
Maybe

```
branches:
  - master
  - main
```

Would be sufficient?

Reply:
It probably would for Safir but I was trying to standardize on a pattern that would also work for packages that had maintenance branches. But maybe that's overkill?